packages/runtime/src/components/shared/grid-item.tsx

```diff
@@ -1,32 +1,45 @@
 import { cx } from '@emotion/css'
-import { ComponentPropsWithoutRef, ElementType } from 'react'
-import { ResponsiveValue, Length as LengthValue } from '../../prop-controllers'
+import { ComponentPropsWithoutRef } from 'react'
+import {
+  ResponsiveValue,
+  Length as LengthValue,
+  ResponsiveNumberValue,
+} from '../../prop-controllers'
 import { useStyle } from '../../runtimes/react/use-style'
+import { useItemAnimation } from '../builtin/Box/animations'
 import { responsiveGridItem } from '../utils/responsive-style'
 
-type BaseProps<T extends ElementType> = {
-  as?: T
+type BaseProps = {
   className?: string
   grid: ResponsiveValue<{ spans: Array<Array<number>>; count: number }>
   index: number
   columnGap?: ResponsiveValue<LengthValue>
   rowGap?: ResponsiveValue<LengthValue>
+  itemAnimateDuration?: ResponsiveNumberValue
+  itemAnimateDelay?: ResponsiveNumberValue
+  itemStaggerDuration?: ResponsiveNumberValue
 }
 
-type Props<T extends ElementType> = BaseProps<T> &
-  Omit<ComponentPropsWithoutRef<T>, keyof BaseProps<T>>
+type Props = BaseProps & Omit<ComponentPropsWithoutRef<'div'>, keyof BaseProps>
 
-export function GridItem<T extends ElementType = 'div'>({
-  as,
```

Comment:
How come we're getting rid of the `as` prop? That didn't seem related to animations specifically

Reply:
I assumed that the as prop / the customizability of the GridItem and the BackgroundContainer were introduced so that we could use `motion.div : 'div'`. Just trying to leave it better than I found it :)

Reply:
Cool, that makes sense. I'm guessing you checked to make sure that no other component using `GridItem` relied on that prop